lib/lor/lib/view.lua

```diff
@@ -39,15 +43,23 @@ function View:render(view_file, data)
         local view_file_name = view_file .. "." .. self.view_ext
 
         local t = template_new(view_file_name)
+
+        local view_func = utils.table_merge(template_helper,self.view_func)
```

Comment:
other developers may have no idea about `template_helper` functions.

if they wanna use some `string` indeed while the inner `template_helper` take its place as a `function`,   they will be confused.

the users themselves should consider about the injected functions.